source/streaming-mode/streaming-read.txt

```diff
@@ -0,0 +1,382 @@
+.. _streaming-read-from-mongodb: 
+
+===================================
+Read from MongoDB in Streaming Mode
+===================================
+
+.. toctree::
+   :caption: Streaming Read Configuration Options
+
+   /streaming-mode/streaming-read-config
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol 
+
+Overview
+--------
+
+When reading a stream from a MongoDB database, the {+connector-long+} supports both 
+*micro-batch processing* and 
+*continuous processing*. Micro-batch processing is the default processing engine, while
+continuous processing is an experimental feature introduced in 
+Spark version 2.3. To learn 
+more about continuous processing, see the `Spark documentation <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html#continuous-processing>`__.
```

Comment:
I got a little confused when reading this overview. I think splitting it up a little more and maybe adding a tiny bit more detail would help. On my first read through I thought micro-batch processing was experimental then got confused why you linked to continuous. Maybe some kind of structure like...

```suggestion
When reading a stream from a MongoDB database, the {+connector-long+} supports both 
**micro-batch processing** and 
**continuous processing**. Micro-batch processing is the default processing engine <that does XYZ>.
Continuous processing is an experimental feature introduced in 
Spark version 2.3 <that does XYZ>. To learn more about continuous processing, see the `Spark documentation <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html#continuous-processing>`__.
```